packages/browser-wallet/src/popup/shared/utils/transaction-helpers.ts

```diff
@@ -204,3 +208,22 @@ export const createPendingTransactionFromAccountTransaction = (
 export function useHasPendingTransaction(transactionType: AccountTransactionType): boolean {
     return useAtomValue(selectedPendingTransactionsAtom).some((t) => t.type === transactionType);
 }
+
+/**
+ * Extract the microCCD amount related for the transaction, excluding the cost.
+ * Note that for many transactions there is no related amount, in which case this returns 0.
+ */
+export function getTransactionAmount(type: AccountTransactionType, payload: AccountTransactionPayload): bigint {
```

Comment:
I suppose scheduled transfer and transfer to encrypted are not here because we don't support them in the first place?

Reply:
Yes, but mostly because their payload types have not been added to the SDK.